### ISSUE TITLE
I am not able to simulate while I am wearing a new ilvl 920 chest piece (Breastplate of the Dark Touch). If I take unequip it, I can sim like usual but as soon as I equip it again, I get this error.

### ISSUE THREAD

**Issue by kairayne** [simulationcraft/simc#3963]
# SimulationCraft encountered an error!
* Category: 
* Error: 

## All options used for simulation:
``` ini
### Begin GUI options ###
item_db_source=local/bcpapi/wowhead
target_error=0
iterations=0
default_world_lag=0.1
max_time=300
vary_combat_length=0.2
fight_style=Patchwerk
tmi_window_global=6
target_level+=3
target_race=Humanoid
default_skill=1.0
optimal_raid=0
override.bloodlust=1
override.bleeding=1
override.mortal_wounds=1
threads=8
process_priority=Low
calculate_scale_factors=1
scale_only=str,crit,haste,mastery,vers
statistics_level=1
### End GUI options ###
### Begin simulateText ###
paladin="Kairaiyne"
level=110
race=blood_elf
region=us
server=dalaran
role=attack
professions=enchanting=115/jewelcrafting=44
talents=1212222
spec=retribution
artifact=2:0:0:0:0:40:1:41:4:42:4:43:4:44:4:47:4:49:1:50:4:51:4:52:4:53:4:54:1:350:1:351:1:352:1:353:1:1118:1:1275:1:1561:4:1562:1:1563:1:1564:6:1647:1

head=,id=138356,bonus_id=3516/1492/3336
neck=,id=141438,enchant_id=5439,bonus_id=3573/1512/3337
shoulder=,id=138362,enchant_id=5441,bonus_id=3516/1497/3336
back=,id=137020,enchant_id=5434,bonus_id=3459/3570
chest=,id=151329,bonus_id=3414/1572/3337
shirt=,id=151116
wrist=,id=145517,bonus_id=3567/1710/1527/3528
hands=,id=145506,bonus_id=3567/1708/1532/3336
waist=,id=137086,bonus_id=1811/3570
legs=,id=145510,bonus_id=3567/1710/1542/3337
feet=,id=148037,gem_id=130220,bonus_id=3579/1707/1808/1537/3528
finger1=,id=137532,bonus_id=3413/1542/3337
finger2=,id=147766,bonus_id=3572/1482/3528
trinket1=,id=139102,bonus_id=3573/604/1572/3528
trinket2=,id=145589,gem_id=130220,bonus_id=3567/1808/1532/3528
main_hand=,id=120978,gem_id=147097/151503/151509,bonus_id=737,relic_id=3561:1487:3336/3567:3149:3336/3567:3154:3337

### Gear from Bags
#
# Ferocious Gladiator's Scaled Helm (900)
# head=,id=145508,gem_id=130220,bonus_id=3567/1709/1808/1532/3337
#
# Fashionable Autumn Cloak (885)
# back=,id=144403,bonus_id=3444/1482/3528
#
# Radiant Lightbringer Cape (885)
# back=,id=147158,bonus_id=3564/40/1467/3528
#
# Cloak of Multitudinous Sheaths (885)
# back=,id=144401,bonus_id=3444/1482/3528
#
# Ferocious Gladiator's Scaled Breastplate (895)
# chest=,id=145502,bonus_id=3567/1708/1527/3528
#
# Legplates of the Highlord (860)
# legs=,id=138359,bonus_id=3520/1457/3528
#
# Iron Ballast Sabatons (905)
# feet=,id=147059,bonus_id=3561/1487/3336
#
# Fierce Combatant's Band (870)
# finger1=,id=149667,bonus_id=3580/1532/3528
#
# Fierce Gladiator's Band (900)
# finger1=,id=148070,bonus_id=3579/1808/1532/3336
#
# Spiked Counterweight (880)
# trinket1=,id=136715,bonus_id=3573/1532/3528
#
# Engine of Eradication (900)
# trinket1=,id=147015,bonus_id=3561/1482/3528
#

### End simulateText ###
desired_targets=1
### Begin overrides ###
# User-specified persistent global and player parameters will be set here.

### End overrides ###
### Begin final options ###
### End final options ###
### END ###
```

**Comment by RyanD** [comment]
While my simulation works, it does treat my 940 Titanforged iron artifact relic as if it does not exist.. imagine my surprise at getting a 40 ilvl upgrade to lose 200k DPS :-D

main_hand=warswords_of_the_valarjar,id=128908,bonus_id=751,gem_id=147089/147114/151292/0,relic_id=3561:1487:3336/3561:1482:3528/3418:1592:3337/0
off_hand=warswords_of_the_valarjar,id=134553

**Comment by vituscze** [comment]
I suspect this has been fixed by updating simc to a version with 7.3 support, if the problem persists, comment here and we'll reopen.
